src/meson.build

```diff
@@ -34,43 +34,37 @@ endif
 executable(
     'dinit',
     dinit_source_files,
-    include_directories : default_incdir,
-    install : true,
-    install_dir : sbindir
+    include_directories: default_incdir,
+    install: true,
+    install_dir: sbindir
 )
-executable(
-    'dinitctl',
-    'dinitctl.cc',
-    'options-processing.cc',
-    include_directories : default_incdir,
-    install : true,
-    install_dir : sbindir
-)
-executable(
-    'dinitcheck',
-    'dinitcheck.cc',
+foreach exec:['dinitctl', 'dinitcheck']
```

Comment:
I think it's better to leave them separate, as it was. Later changes may add source files to one or the other and if you combine them now you just have to split them then, i.e. refactoring should not combine things that are _coincidentally_ the same.

You can use a `kwargs` argument and map with `include_directories`, `install` and `install_dir` values if you really want to remove some duplication.